cmd/controller-manager/main.go

```diff
@@ -217,9 +221,8 @@ func main() {
 				OnStoppedLeading: onStopped,
 			},
 		})
-	}, cliCfg.WaitDuration)
+	}()
 
-	srv := createHTTPServer()
 	sc := make(chan os.Signal, 1)
 	signal.Notify(sc,
 		syscall.SIGHUP,
```

Comment:
I don't think we should close  `sc` at line 237, if there are more signal it will panic to push into `sc` before quit

Reply:
removed in https://github.com/pingcap/tidb-operator/pull/3890/commits/915fb4e9127396601a9aa45f45a795f8a10e49c6.